tasks.py

```diff
@@ -43,6 +41,7 @@ def bin_prefix(cmd):
 @task
 def server(host=None, port=5000, debug=True, live=False):
     """Run the app server."""
+    from website import settings
```

Comment:
But setting is already imported in `website.app`; why does every task need to import it?